scripts/packages/chocolatey/bazel.nuspec.template

```diff
@@ -75,8 +75,9 @@ Supply like `--params="/option:value ..."` ([see docs for --params](https://gith
       <dependency id="chocolatey-uninstall.extension" />
     </dependencies>-->
     <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.0.7"/>
       <dependency id="jdk8" version="[8.0.102,)"/>
-      <dependency id="msys2" version="[20160719.1,)"/>
+      <dependency id="msys2" version="[20160719.1.0]"/>
```

Comment:
In that case, shall I simply back out this change (the version pin) as unnecessary?